packages/vscode-js-profile-core/src/open-location.ts

```diff
@@ -102,6 +122,23 @@ const showPositionInUrl = async (
   await showPosition(document, lineNumber + 1, columnNumber + 1, viewColumn);
   return true;
 };
+/**
+ * Parses a link into a link object
+ * @param url
+ * @returns
+ */
+const parseLink = (link: string | undefined): Link => {
+  const matchCommand = link?.match(/^command:([\w\.]+)(?:\?(.*))?/);
+  if (matchCommand) {
+    const [command, rawArgs] = matchCommand.slice(1);
+    const parsed = rawArgs ? JSON.parse(decodeURIComponent(rawArgs)) : [];
+    const args = Array.isArray(parsed) ? parsed : [parsed];
+    return { type: LinkType.Command, command, args };
+  }
+  if (link?.match(/\w\w+:/))
```

Comment:
This is missing a `^` to ensure it matches the prefix

```suggestion
  if (link?.match(/^\w{2,}:/))
```

Reply:
yep, well spotted